src/View/Form/FormContext.php

```diff
@@ -135,8 +135,9 @@ public function hasError($field)
     public function error($field)
     {
         $errors = $this->_form->errors();
-        if (isset($errors[$field])) {
-            return array_values($errors[$field]);
+        $errors = \Cake\Utility\Hash::extract($errors, $field);
```

Comment:
Sure, I just started with the patch I posted in #7532. I need a little time to clean things up a bit.